sound/soc/sof/topology.c

```diff
@@ -2508,18 +2437,6 @@ static int sof_widget_unload(struct snd_soc_component *scomp,
 			list_del(&dai->list);
 		}
 		break;
-	case snd_soc_dapm_scheduler:
-		mutex_lock(&sdev->cores_status_mutex);
-
-		/* power down the pipeline schedule core */
-		pipeline = swidget->private;
-		ret = snd_sof_dsp_core_power_down(sdev, 1 << pipeline->core);
```

Comment:
presumably this is now done for all widgets / components?

Reply:
yes.